sklearn/manifold/_spectral_embedding.py

```diff
@@ -483,17 +483,17 @@ class SpectralEmbedding(BaseEstimator):
     References
     ----------
 
-    - A Tutorial on Spectral Clustering, 2007
+    - :doi:`A Tutorial on Spectral Clustering, 2007
       Ulrike von Luxburg
-      http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.165.9323
+      <10.1007/s11222-007-9033-z>`
 
-    - On Spectral Clustering: Analysis and an algorithm, 2001
+    - :doi:`On Spectral Clustering: Analysis and an algorithm, 2001
       Andrew Y. Ng, Michael I. Jordan, Yair Weiss
-      http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.19.8100
+      <10.5555/2980539.2980649>`
```

Comment:
I can't find a DOI for this one, please revert, as this can't be anything else than an external link